### ISSUE TITLE
'main' panicked , could not open input rad file

### ISSUE THREAD

**Issue by Varix** [COMBINE-lab/alevin-fry#23]
Hey all, following this tutorial , https://combine-lab.github.io/alevin-fry-tutorials/2021/alevin-fry-velocity/ after someone at BioC Conference recommended it as a good way to do pre-processing for velocity analysis. While salmon is my goto tool for BulkSeq , I must admit I have very little experience with sister alevin for single cell as most of our data is 10X so have stayed in the STAR/Cellranger ecosystem.

everything works in the tutorial until this line:

`alevin-fry generate-permit-list -d fw -k -i sample_v2_map -o sample_v2_quant`

and then I get this error:

`thread 'main' panicked at 'could not open input rad file: Os { code: 2, kind: NotFound, message: "No such file or directory" }', libradicl/src/cellfilter.rs:194:52`

quick google search didnt find any hits , hoping it was a simple configuration issue.

alevin-fry 0.1.0
Python 3.8.5
Ubuntu 20.04 LTS

**Comment by rob-p** [comment]
Hi @Varix,

  Thanks for the report!  I'd recommend first updating to the most recent version of fry (should be easy using conda or biocontainers).  However, the panic seems to result from an expected input file not being found.  Could you share the result of an `ls` on `sample_v2_map`?

Thanks!
Rob

**Comment by Varix** [comment]
thanks for the reply Rob, I meant to come back here and post updates. 

I was able to solve the issue with a very similar solution to your recommendation. I had an older version of salmon from my bulkseq days and when I ran `salmon index -t transcriptome_splici_fl146.fa -i splici_idx -p 27`  , it worked of course , but produced an output data structure that my fry version was not expecting.... i installed the newest version of `salmon` in a new conda environment , rebuilt the index with the same commands and then ran `alevin-fry ` and the rest of the tutorial worked as intended. 

I do have a whole bunch of other questions but maybe I can find your in another forum/slack/discord and pick your brain on those questions at another time.

thanks for contributing to this great pipeline
